src/lib/src/charts/stacked-bar-category-rotated-chart/stacked-bar-category-rotated-chart.component.ts

```diff
@@ -1,63 +0,0 @@
-import { Component, Input, OnChanges, Inject, ElementRef } from '@angular/core';
```

Comment:
I dropped support for stacked rotated bar charts. They came up once early on with playback data but didn't go over well, so the component hasn't been kept up to date anyway. We can still bring them back if we decide we want them later. The hard part was figuring out the C3 chart configuration.

Reply:
Sounds good - git history ftw!